### ISSUE TITLE
Only complete URLs can be added as a link within the Android app

### ISSUE THREAD

**Issue by matkv** [omnivore-app/omnivore#3070]
When trying to add a link with the "+" button, only full links are saved.

For example, trying to add "example.com" doesn't work , only "https://example.com" works.

Seems like it would be more user friendly to allow the first option as well.

**Comment by remychantenay** [comment]
Thanks for reporting this 👍

Just to add more details, the behavior is currently consistent with iOS. I.e., the scheme (https) is mandatory.

I suppose we could automatically prepend the host with the scheme if not present.

But I am wondering how often one would manually input an URL w/o the scheme 🤔 I personally always copy/paste and the scheme is always present.

**Comment by matkv** [comment]
> But I am wondering how often one would manually input an URL w/o the scheme 🤔 I personally always copy/paste and the scheme is always present.

Yeah same, but then there should be some sort of visual feedback that it isn't possible I guess. Currently I think it just lets you press the "add" button and the dialog closes without saving anything but the successfully saved message still comes up.
